test/lsp/lsp_test.bzl

```diff
@@ -48,3 +55,26 @@ def update_test():
         tags = ["manual"],
         tests = update_rules,
     )
+
+_PROTOCOL_TEST_SRCS = ["ProtocolTest.cc", "ProtocolTest.h"]
+
+def protocol_tests(cc_files):
+    for cc_file in cc_files:
+        native.cc_test(
+            name = dropExtension(basename(cc_file)),
+            size = "medium",
```

Comment:
Not at all blocking, but do these tests pass with `timeout = "moderate"` instead of `size = "medium"`?

https://bazel.build/reference/be/common-definitions#test.timeout

Reply:
These pass on my machine with `timeout = "moderate"` and `-c dbg`, so I think we are good here!